### ISSUE TITLE
Offset don't work

### ISSUE THREAD

**Issue by keldi23** [rdmtc/node-red-contrib-sun-position#119]
Hello,

when I choose "Sunset End" with an offset in minutes, it don't work. Especially over an offset of 20.
When I change the offset from 20 to 30 minutes it will change from 19:02 to 19:42.

Best Regards
Stefan

**Comment by Hypnos3** [comment]
Hi there,

Would it be possible for you to tell me where you see this behavior?

The collection now contains 8 nodes where multiple times can be entered an Offset, used in execution and in several places used for display times.

Thanks & Greetings
Robert

**Comment by keldi23** [comment]
Hello,

thanks for your answer.
I want to take some screenshots today.  I just had some other Bugs.
After full deploy everything was ok. So I think it was a problem of Node-Red.

Thank you

**Comment by keldi23** [comment]
Hello,

the problem isn't solved.
I have a node "inject at time". 
Time "Sonnenuntergang Ende" with offset "Number" 20 Minutes.
There ist one trigger at the time witch I can see at the node. 19:18 o clock.
But the problem is a second trigger 20 minutes later at 19:38.
How can I solve the problem?

Greets

**Comment by Hypnos3** [comment]
Unfortunately I could not see that behavior. Could you please export your inject node?

**Comment by keldi23** [comment]
Hello,

here is my node



[flows.txt.zip](https://github.com/rdmtc/node-red-contrib-sun-position/files/4396757/flows.txt.zip)

**Comment by morres83** [comment]
I can confirm that there is a bug with "inject at time".
Setup:
1x inject at blue morning, offset 40 min - for action: open door
1x inject at blue evening, no offset; - for action: close door

Behavior:
Signal at 6:00am ->open door -> OK
Signal at 7:26pm -> close door -> OK
Signal at 8:13pm -> open door -> not OK
Signal at 8:40pm -> open door -> not OK
At this point I realized the malfunction (because I get Telegram messages), and I checked node-red. The NEXT expected time for "open door" as shown in the flow below the "inject at time" block was 8:40pm NEXT day, which is obviously also wrong.

BR,
 morres
